app/controllers/component-tree.js

```diff
@@ -218,6 +218,14 @@ class RenderItem {
     return this.renderNode.name;
   }
 
+  get args() {
+    return this.renderNode.args;
+  }
+
+  get isCurlyInvocation() {
+    return this.renderNode && this.renderNode.args && this.renderNode.args.positional;
```

Comment:
```suggestion
    return this.renderNode.args && this.renderNode.args.positional;
```

`this.renderNode` is guaranteed